### PR TITLE
Practicing GitHub Flow For Class - kna

### PR DESCRIPTION
**Pull request by kesavan2**
### TODO

- [x] get review from individuals 
- [ ] actually include some content 

@allthedoll your thoughts?

:8ball: 

If you haven't already, please comment on #3

### CODE REVIEW

File: kna_hello_world.jql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#Example of finding an Issue's SubTasks
```
Comment (itsmyway) [inline kna_hello_world.jql]: What lang is this code?

Comment (itsmyway) [inline kna_hello_world.jql]: What lang is this code?

Comment (kesavan2) [inline kna_hello_world.jql]: Just JQL for JIRA search syntax

**Comment by kesavan2** [comment]
Deleting branch to cleanup after successful merge of branch

**Pull request merged**